internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -1107,7 +1107,7 @@ func newAssistedCM(ctx context.Context, log logrus.FieldLogger, asc ASC) (client
 			"ISO_CACHE_DIR":          "/data/cache",
 
 			// from configmap
-			"AUTH_TYPE":                   "local",
+			"AUTH_TYPE":                   "none",
```

Comment:
is that temporary for testing?

Reply:
Yeah, it's just a quick POC for testing purposes.